File: bootpay/src/main/java/kr/co/bootpay/listener/EventListener.java
package kr.co.bootpay.listener;


public interface EventListener extends CancelListener, ErrorListener, DoneListener, CloseListener,  ReadyListener, ConfirmListener {
    void onError(String data);

    void onCancel(String data);

    void onClose(String data);

    void onReady(String data);

    void onConfirm(String data);

    void onDone(String data);
}

File: bootpay/src/main/java/kr/co/bootpay/listner/EventListener.kt
package kr.co.bootpay.listner

interface EventListener : CancelListener, ErrorListener, DoneListener, CloseListener,  ReadyListener, ConfirmListener {
    override fun onError(message: String?)

    override fun onCancel(message: String?)

    override fun onClose(message: String?)

    override fun onReady(message: String?)

    override fun onConfirm(message: String?)

    override fun onDone(message: String?)
}
